package.json

```diff
@@ -57,11 +57,15 @@
     "braid-design-system": ">= 31.21.0",
     "react": ">= 17 < 19",
     "react-router-dom": ">= 5.3.0",
-    "sku": ">= 10.13.4 < 13"
+    "sku": ">= 10.13.4 < 13",
+    "@mdx-js/loader": "^1.6.22"
   },
   "peerDependenciesMeta": {
     "@mermaid-js/mermaid-cli": {
       "optional": true
+    },
+    "@mdx-js/loader": {
```

Comment:
Could consumers run into TypeScript errors if this isn't installed and they run `sku lint` with `skipLibCheck: false`?

Reply:
Ah, looks like it's only referenced in a JS Webpack file 🙏